### DISCUSSION TITLE
How to use ParaView to extract contour lines and output data along the path of .e files

### DISCUSSION THREAD

**Discussion by DP52** [peridigm/peridigm#310] [General]
In the post-processing process, I found that ParaView could get contour lines and extract data along the path when processing other types of files, but could not get any data when doing the same operation for .e files. As shown in the figures below, the first picture is the contour result obtained by post-processing the .foam file with ParaView. The second image is a graph along the path extracted from the .ex2 file.

[
![image](https://user-images.githubusercontent.com/128480691/226619505-db3bc09f-f512-44ae-8696-e5b76002c5a4.png)
](url)

[
![image](https://user-images.githubusercontent.com/128480691/226619569-33cdbe2d-a088-4cde-93ab-ec9087f4b800.png)
](url)

I would like to know how to use paraview to post-process the .e file to get the contour line and how to get the graph along the path, thank you!

**Comment by djlittl** [comment]
I think the issue is that if you have a data file that contains finite elements, then ParaView can interpolate between the points using simple finite element interpolation rules.  The data from Peridigm is basically a point cloud, so ParaView has no way of interpolating between points.  The ability to interpolate is necessary for some post-processing operations, and I think that’s what’s happening here with the contour plots.  The .e file format is not the issue.  If you had finite element data in a .e file, ParaView would be able to make a contour plot.
